pkg/helpers/apply/applierbuilder.go

```diff
@@ -76,3 +85,12 @@ func (a *ApplierBuilder) WithOwner(owner runtime.Object, blockOwnerDeletion, con
 	a.scheme = scheme
 	return a
 }
+
+func (a *ApplierBuilder) WithCache(cache resourceapply.ResourceCache) *ApplierBuilder {
```

Comment:
you might want to comment in what cases this should be used.

Reply:
Ok